set.mm

```diff
@@ -706039,6 +706114,56 @@ not even needed (it can be any class).  (Contributed by Glauco
       ( wcel wral wf ralrimia fmptff sylib ) ADELZBCMCEFNARBCGJOBCEDFHIKPQ $.
   $}
 
+  ${
+    fvmpt2df.1 $e |- F/_ x A $.
+    fvmpt2df.2 $e |- F = ( x e. A |-> B ) $.
+    fvmpt2df.3 $e |- ( ( ph /\ x e. A ) -> B e. V ) $.
+    $( Deduction version of ~ fvmpt2 .  (Contributed by Glauco Siliprandi,
+       24-Jan-2025.) $)
+    fvmpt2df $p |- ( ( ph /\ x e. A ) -> ( F ` x ) = B ) $=
+      ( cv wcel wa cfv cmpt fveq1i wceq id fvmpt2f syl2an2 eqtrid ) ABJZCKZLUAE
+      MUABCDNZMZDUAEUCHOUBUBADFKUDDPUBQIBCDFGRST $.
+  $}
+
+  ${
+    $d A a t v w x y z $.  $d J a t w x y z $.  $d V a t w x y z $.
+    rn1st.1 $e |- F/_ x B $.
+    $( The range of a function with a first-countable domain is itself
+       first-countable.  This is a variation of ~ 1stcrestlem , with a
+       essential non-free condition replacing a disjoint variable constraint.
```

Comment:
a essential -> an essential

or "essential" can be removed

Reply:
[c0aa5ae](https://github.com/metamath/set.mm/pull/4606/commits/c0aa5aec1814cf659f66d18bf76d1608ee5b7804) should have fixed this